app/models/katello/system.rb

```diff
@@ -5,15 +5,12 @@ class System < Katello::Model
     self.include_root_in_json = false
 
     include Hooks
-    define_hooks :add_host_collection_hook, :remove_host_collection_hook,
-                 :add_activation_key_hook, :remove_activation_key_hook,
-                 :as_json_hook
+    define_hooks :as_json_hook
```

Comment:
I can't recall what this did exactly, and I don't see it being used anywhere. We may have included this once as a way to add your own attributes to the JSON representation of the system but I don't see that being as useful with the way RABL works. I think that this may be the only instance of the hook usage left as well, which would mean we could drop another gem dependency if we removed this (but please double check that).